src/main/java/uk/gov/ida/verifyserviceprovider/services/AssertionDataTranslator.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.ida.verifyserviceprovider.services;
```

Comment:
Is this file necessary?

I'm actually a fan of having interfaces for everything, but that's not a pattern we seem to be keen on in Verify, and this one isn't being referenced anywhere (yes, one class extends it, but there are no references to the interface as a type).

Reply:
Maybe it isn't needed